website/docs/r/private_dns_glb_pool.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : dns_glb_pool"
+sidebar_current: "docs-ibm-resource-dns-glb-pool"
+description: |-
+  Manages IBM Private DNS glb pool.
+---
+
+# ibm\_dns_glb_pool
+
+Provides a private dns glb pool resource. This allows dns glb pool to be created,updated and deleted.
+
+## Example Usage
+
+```hcl
+
+resource "ibm_dns_glb_pool" "test-pdns-glb-pool-nw" {
+    instance_id = ibm_resource_instance.test-pdns-instance.guid
+    vpc_crn = ibm_is_vpc.test_pdns_vpc.crn
+    type = "vpc"
+    description = "new nimi pool"
+    enabled="true"
+    healthy_origins_threshold=1
+    origins {
+	    name    = "example-1"
+     	address = "www.google.com"
+	    enabled = true
+	    description="origin pool"
+    }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, string) The id of the private DNS on which zone has to be created.
+* `name` - (Required, string) Name of the load balancer pool.
+* `description` -  (string) Descriptive text of the load balancer pool.
+* `enabled` -  (bool) Whether the load balancer pool is enabled.
+* `healthy_origins_threshold` -  (string) The minimum number of origins that must be healthy for this pool to serve traffic. If the number of healthy origins falls below this number, the pool will be marked unhealthy and we will failover to the next available pool.
+* `origins` -  (Required, string) The list of origins within this pool. Traffic directed at this pool is balanced across all currently healthy origins, provided the pool itself is healthy.
+* `name` -  (string) The name of the origin server.
+* `description` -  (string) Description of the origin server.
```

Comment:
add it is optional or required. this comment applicable to all the attributes

Reply:
Done